Bottom: 089fd8db0fb871c7ca2e99ac7f4311e9010432ca
Top:    7995ce65c762cc782885f999be1655c53385b187
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-16 04:08:30 -0300

Refresh of manual-document-mt-safety-crypt.patch

---

diff --git a/manual/crypt.texi b/manual/crypt.texi
index 5c9f6f7..a3c8fa5 100644
--- a/manual/crypt.texi
+++ b/manual/crypt.texi
@@ -99,6 +99,7 @@ in a convenient way.
 @c then restore the terminal mode.  It has a cleanup to close the stream
 @c in case of (synchronous) cancellation, but not to restore the
 @c terminal mode.
+
 @code{getpass} outputs @var{prompt}, then reads a string in from the
 terminal without echoing it.  It tries to connect to the real terminal,
 @file{/dev/tty}, if possible, to encourage users not to put plaintext
